### PR TITLE
remove support for Python 2.6 and 3.2

### PR DESCRIPTION
**Pull request by ezarowny**
Requests 2.11.0 is not compatible with Python 3.2!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -74,7 +74,7 @@
         "Topic :: System :: Monitoring",
         ],
     install_requires=[
-        'requests<=2.10.0',
+        'requests>=0.12.1,<=2.10.0',
```
Comment (brianr) [inline setup.py]: Hm. Can we have the `<=2.10.0` part only present for the python versions where this is needed?

Comment (ezarowny) [inline setup.py]: Actually, yeah I can do that. I've already done some conditional stuff for the tests_require part.

File: setup.py [hunk 2]
```diff
@@ -17,14 +17,25 @@
...
 tests_require = [
     'mock',
     'webob',
     'blinker',
     'unittest2'
-    ]
+]
 
 version = sys.version_info
+if version[0] == 3 and version[1] == 2:
```
Comment (brianr) [inline setup.py]: can you add a comment explaining this?

File: setup.py (hunk 1 above)
Comment (coryvirok) [inline setup.py]: I don't think we should only do the `<=` for python versions where this is needed. Doing so exposes us to any breaking changes that the requests maintainers make. 

E.g. A user is on Python 3.1 and we are not specifying `<=` for the requests package. Then, the requests maintainers add some functionality which breaks things for Python 3.1. We will have no way of knowing that this happens for our customers unless we re-run Travis tests. 

By restricting the range to what we know works right now we are playing it safe for future changes made to the requests library.

Comment (brianr) [inline setup.py]: @coryvirok But wouldn't `<=` mean that if you are using pyrollbar, you can't upgrade requests (for the purposes of the rest of your application) to a version beyond that? That seems worse to me.

Comment (coryvirok) [inline setup.py]: Ah. That is a good point. Then the correct behavior is probably to require >= the min version and throw an exception and disable if the wrong version of requests is used.

**Pull request merged**